bpf/cpu/common.h

```diff
@@ -27,6 +26,22 @@
 typedef struct {
   u64 len;
   u64 addresses[MAX_STACK_DEPTH];
-} stack_trace_t;
+} Stack;
+// NOTICE: stack_t is defined in vmlinux.h.
+
+#define CLASS_NAME_MAXLEN 32
+#define METHOD_MAXLEN 64
+#define PATH_MAXLEN 128
 
+typedef struct {
+  char class_name[CLASS_NAME_MAXLEN];
+  char method_name[METHOD_MAXLEN];
+  char path[PATH_MAXLEN];
+  u32 lineno;
```

Comment:
Nice! For the first iteration we can keep it like this (it's the same for rbperf) but later on we can change this to not have the line number and store it in the upper / lower 32 bits of the stack